tools/cockpit.spec

```diff
@@ -444,12 +429,40 @@ fi
 %systemd_preun cockpit.socket cockpit.service
 
 %postun ws
-if [ -x %{_sbindir}/selinuxenabled ]; then
-    %selinux_modules_uninstall -s %{selinuxtype} %{name}
-    %selinux_relabel_post -s %{selinuxtype}
-fi
 %systemd_postun_with_restart cockpit.socket cockpit.service
 
+%package selinux-policies
+Summary: selinux policies required by cockpit
+# older -ws contained the SELinux policy, now split out
+Conflicts: %{name}-ws < 337-1.2025
+Requires(post): selinux-policy-%{selinuxtype} >= %{selinux_policyver}
+%if 0%{?suse_version}
+Requires(post): selinux-tools
+%else
+Requires(post): libselinux-utils
+%endif
```

Comment:
The SUSE-specific conditional is not required, since `libselinux-utils` works on both platforms.